doc/overview.rst

```diff
@@ -676,12 +676,3 @@ For comprehensive examples on how to use Inference Pipelines please refer to the
 - `inference_pipeline_sparkml_xgboost_abalone.ipynb <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/inference_pipeline_sparkml_xgboost_abalone/inference_pipeline_sparkml_xgboost_abalone.ipynb>`__
 - `inference_pipeline_sparkml_blazingtext_dbpedia.ipynb <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/inference_pipeline_sparkml_blazingtext_dbpedia/inference_pipeline_sparkml_blazingtext_dbpedia.ipynb>`__
 
-
```

Comment:
why are we removing this part?

Reply:
If you mean the above, it's just an extra blank line.

If you mean Workflow, it hasn't been migrated yet, so it needs to stay on the GitHub page, not in overview.rst

Reply:
I think it'd be better to keep the Workflow section but replace the link with an absolute link to the GH readme, and then update that link once the Workflow readme has been migrated

Reply:
Fair enough. I can migrate workflow next, so that it is there before the next release.